### PR TITLE
UIOR-228-Capture instance uuid and product id's when user selects instance

### PR DESCRIPTION
**Pull request by AndreiShumski2**
## Purpose
Capture instance uuid and product id's when user selects instance
https://issues.folio.org/browse/UIOR-228

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/OrdersList/OrdersList.js [hunk 1]
```diff
@@ -98,6 +98,12 @@ class OrdersList extends Component {
       records: 'mtypes',
       perRequest: 1000,
     },
+    identifierTypes: {
+      type: 'okapi',
+      path: 'identifier-types',
+      records: 'identifierTypes',
+      perRequest: 1000,
```
Comment (aliaksei-chumakou) [inline src/OrdersList/OrdersList.js]: please use another constant, with max value supported by back-end

File: src/components/POLine/Item/ContributorForm.js [hunk 2]
```diff
@@ -64,14 +70,18 @@ class ContributorForm extends Component {
             fullWidth
             label={<FormattedMessage id="ui-orders.itemDetails.contributor" />}
             name={`${elem}.contributor`}
+            onChange={(e) => this.props.onChangeField(e.target.value, `${elem}.contributor`)}
```
Comment (aliaksei-chumakou) [inline src/components/POLine/Item/ContributorForm.js]: I think `(e, value) => ` should work

File: translations/ui-orders/en.json [hunk 3]
```diff
@@ -391,18 +391,7 @@
   "itemDetails.description": "Description",
   "itemDetails.edition": "Edition",
   "itemDetails.productId": "Product ID",
-  "itemDetails.productIdType.CODEN": "CODEN",
```
Comment (aliaksei-chumakou) [inline translations/ui-orders/en.json]: Are you sure we don't need to translate these values anymore?

File: src/OrdersList/OrdersList.js (hunk 1 above)
Comment (AndreiShumski2) [reply src/OrdersList/OrdersList.js]: fixed

File: src/components/POLine/Item/ContributorForm.js (hunk 2 above)
Comment (AndreiShumski2) [reply src/components/POLine/Item/ContributorForm.js]: Fixed

File: translations/ui-orders/en.json (hunk 3 above)
Comment (AndreiShumski2) [reply translations/ui-orders/en.json]: Asked in Jira

**Pull request closed without merging**